src/xmagics/executable.cpp

```diff
@@ -15,6 +15,10 @@
 #include <string>
 #include <vector>
 
+#if defined(WIN32)
```

Comment:
Well the cling and clang framework produce a lot of warnings in a Visual Studio build, which is very uggly as you don't see your own warnings anymore.

The following lines will disable the warnings completly. I think this is fine as it only for the external library.